clippy.toml

```diff
@@ -0,0 +1 @@
+allow-unwrap-in-tests = true
```

Comment:
I have added this file to configure clippy.

Reply:
May be a `#![allow(clippy::unwrap_used, clippy::expect_used)]` directive in test files would be better?

Reply:
We can do that. It might be better because clippy config file like `clippy.toml` is unstable.

Reply:
Fixed
[889d250](https://github.com/SeaQL/sea-orm/pull/2056/commits/889d250d54663f0b35fcdb3c1981ba1b820a252c)